R/compile-dll.r

```diff
@@ -108,12 +108,33 @@ needs_compile <- function(pkg = ".") {
   source > dll
 }
 
+makevars <- function(pkg = ".") {
+  pkg <- as.package(pkg)
+  srcdir <- file.path(pkg$path, "src")
+  dir(srcdir, "^Makevars.*$", recursive = TRUE, full.names = TRUE)
```

Comment:
I think this would be more convenient if it returned the lines (or `character()`) if no makefile